docs/src/literate-tutorials/ns_vs_diffeq.jl

```diff
@@ -1,3 +1,7 @@
+# Putting this flag to false reproduces the figure shown in the example #src
+# We check for laminar flow development in the CI                       #src
+IS_CI = true #hide
```

Comment:
https://docs.julialang.org/en/v1/base/base/#Base.@isdefined
```suggestion
if !(@isdefined is_ci) #hide
    is_ci = false
end
```

Maybe something like this? @fredrikekre

Reply:
so we define `is_ci` in the beginning of the CI execution ?

Reply:
https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/docs/make.jl#L15C1-L15C44

Reply:
ah so I can change it already?